### PR TITLE
Bamboo spaces

### PR DESCRIPTION
**Pull request by Ibai-NexxyoLabs**
Added custom skin and custom domain for BambooDeFi

### CODE REVIEW

File: spaces/domains.json [hunk 1]
```diff
@@ -123,5 +123,6 @@
   "gov.premia.finance": "premia.eth",
   "vote.perlinx.finance": "perlinx.eth",
   "vote.defigeek.xyz": "defigeek.eth",
-  "ambassador.synthetix.io": "snxambassador.eth"
+  "ambassador.synthetix.io": "snxambassador.eth",
+  "bamboodefi.com": "bamboodefi.eth"
```
Comment (gokaiorg) [inline spaces/domains.json:127]: Are you sure you want to point your Snapshot space to your main domain and not create a sub-domain instead? For example:
```suggestion
  "vote.bamboodefi.com": "bamboodefi.eth"
```

File: skins/bamboodefi.scss [hunk 2]
```diff
@@ -0,0 +1,10 @@
+.bamboodefi {
+  --primary-color: #7ac949;
+  --bg-color: white;
+  --text-color: black;
+  --link-color: #111111;
+  --heading-color: #111111;
+  --border-color: black;
+  --header-bg: white;
+  --block-bg: transparent;
+}
```
Comment (gokaiorg) [inline skins/bamboodefi.scss:10]: The colors used in your skin make the interface unreadable, please change with others or remove the skin to use the default theme.
image

**Review by gokaiorg — CHANGES_REQUESTED** [review]
I can't find your new space. Please make sure that you have filled in all required fields and added a strategy.
https://snapshot.page/#/bamboodefi.eth

File: spaces/domains.json (hunk 1 above)
Comment (Ibai-NexxyoLabs) [reply spaces/domains.json]: Yes, vote.bamboodefi.com is good.

File: skins/bamboodefi.scss (hunk 2 above)
Comment (Ibai-NexxyoLabs) [reply skins/bamboodefi.scss:10]: The link color attribute has been changed

File: spaces/domains.json [hunk 3]
```diff
@@ -124,5 +124,6 @@
   "vote.perlinx.finance": "perlinx.eth",
   "vote.defigeek.xyz": "defigeek.eth",
   "ambassador.synthetix.io": "snxambassador.eth",
+  "vote.bamboodefi.com": "bamboodefi.eth"
```
Comment (gokaiorg) [inline spaces/domains.json:127]: You miss a comma at the end of the line.
```suggestion
  "vote.bamboodefi.com": "bamboodefi.eth",
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]
Your space still not loading. Do you need some help to create it?

**Comment by gokaiorg** [comment]
@Ibai-NexxyoLabs You may miss to set a name before saving your space settings, cause your space still not loading properly and you have a name error in the console.
image

**Review by gokaiorg — CHANGES_REQUESTED** [review]
.

File: skins/bamboodefi.scss [hunk 4]
```diff
@@ -0,0 +1,10 @@
+.bamboodefi {
+  --primary-color: #7ac949;
+  --bg-color: white;
+  --text-color: black;
+  --link-color: #ffffff;
+  --heading-color: #111111;
+  --border-color: black;
+  --header-bg: white;
+  --block-bg: transparent;
+}
```
Comment (gokaiorg) [inline skins/bamboodefi.scss:10]: The colors used in your skin make the interface unreadable, please change with others or remove the skin to use the default theme.
image

**Review by gokaiorg — CHANGES_REQUESTED** [review]

**Review by Ibai-NexxyoLabs — COMMENTED** [review]
The changes have been added

**Review by Ibai-NexxyoLabs — COMMENTED** [review]
CSS styles fixed

**Review by gokaiorg — APPROVED** [review]

**Pull request merged**